schema/bundle_header.sql

```diff
@@ -24,5 +24,5 @@ CREATE TABLE IF NOT EXISTS mev.bundle_header
     `express_lane_round` Nullable(UInt64)
 ) 
 ENGINE = MergeTree()
-PRIMARY KEY (`block_number`, `tx_hash`)
-ORDER BY (`block_number`, `tx_hash`)
\ No newline at end of file
+PRIMARY KEY (`timeboosted`, `express_lane_round`, `block_number`, `tx_hash`)
```

Comment:
since the indexing might be used in other places, can we manually create additional index?

and why do we need indexing for four different columns at once?